.github/workflows/smoke-test-pr-check.yaml

```diff
@@ -0,0 +1,156 @@
+#
+# Copyright (c) 2023 Red Hat, Inc.
+# This program and the accompanying materials are made
+# available under the terms of the Eclipse Public License 2.0
+# which is available at https://www.eclipse.org/legal/epl-2.0/
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+# Contributors:
+#   Red Hat, Inc. - initial API and implementation
+#
+
+name: Smoke Test
+
+# Trigger the workflow on pull request
+on:
+  pull_request:
+    paths-ignore:
+      - '**/*.md'
+      - devfile.yaml
+      - '.che/*'
+      - '.rebase/*'
+env:
+  LOCAL_TEST_DIR: /tmp
+jobs:
+  smoke-test:
+    runs-on: ubuntu-22.04
+    steps:
+
+      - name: Checkout che-code source code
+        uses: actions/checkout@v2
+
+      - name: Configuring nodejs 14.x version
+        uses: actions/setup-node@v1
+        with:
+          node-version: '14.x'
```

Comment:
```suggestion
          node-version: '16'
```

Reply:
It should fix the warning:
![image](https://github.com/che-incubator/che-code/assets/1636395/a8a284bc-db60-4f4a-93ab-b2e0401acfc6)

Reply:
Done